src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -48,11 +45,21 @@
       + "watch\\?v%3D|watch\\?feature=player_embedded&v=|%2Fvideos%2F|"
       + "embed%\u200C\u200B2F|youtu.be%2F|%2Fv%2F)[^#\\&\\?\\n]*";
 
+  @VisibleForTesting static final String PARAM_ID = "id";
+
   /* Name of input field used for lecture name in lecture selection page. */
-  private static final String NAME_INPUT = "name-input";
+  @VisibleForTesting static final String PARAM_NAME = "name-input";
```

Comment:
This can be public.

Reply:
"If you expose constants in Java, please mark the modifications with @VisibleForTesting"
This is from the doc: https://docs.google.com/document/d/18T9ef-DRqTncVD377GoIQmOGI84cNKkZ7Nn07YURFTw/edit